quill-cassandra/src/main/scala/io/getquill/source/cassandra/ClusterSession.scala

```diff
@@ -0,0 +1,22 @@
+package io.getquill.source.cassandra
+
+import com.typesafe.config.Config
+import com.datastax.driver.core.Cluster
+
+object ClusterSession {
+
+  def apply(config: Config) = {
+    val builder = Cluster.builder
```

Comment:
there are other important options that users should be able to set through a config file, i.e. consistency level, retries, load balancing policies, ...